pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -72,6 +76,19 @@ func (plugin *kubenetNetworkPlugin) Init(host network.Host) error {
 		glog.Warningf("Failed to find default bridge MTU: %v", err)
 	}
 
+	// Set bridge-nf-call-iptables=1 to maintain compatibility with older
```

Comment:
I'd update this comment to say something like:

// Since this plugin uses a Linux bridge, set bridge-nf-call-iptables=1
// is necessary to ensure kube-proxy functions correctly.

There isn't any backwards compat concerns here really since kubenet is something the user explicitly selects.